### PR TITLE
LPS-98877

### PR DESCRIPTION
**Pull request by samuelkong**
Brian wanted this to be sent to you first. See https://github.com/brianchandotcom/liferay-portal/pull/76261#issuecomment-516072464

@arthurchan35

### CODE REVIEW

File: modules/core/portal-bootstrap/system.packages.extra.bnd [hunk 1]
```diff
@@ -130,7 +130,7 @@ Export-Package:\
 	\
 	javax.persistence.*;version='2.0',\
 	\
-	javax.portlet.*;version='2.0',\
+	javax.portlet.*;version='3.0.1',\
```
Comment (shuyangzhou) [inline modules/core/portal-bootstrap/system.packages.extra.bnd]: @samuelkong this is not right, we are forcing to provide 2.0 api, doing so could cause some portlets not resolving. What's the reason for this change?

Comment (arthurchan35) [reply modules/core/portal-bootstrap/system.packages.extra.bnd]: @shuyangzhou ,
As far as I remember, the tests I have done before showed that if we dont provide 3.0.1, then some portlets will complain unresolved.
In fact, we can simply run ci test to see if it's providing the correct portlet api version.
I mean, in the dependencies.properties, the actually portlet-api being used is 3.0.1.

**Comment by arthurchan35** [comment]
ci:test:relevant

**Comment by arthurchan35** [comment]
Seems like there are rebasing conflicts during last two days
but it's the same tests from https://github.com/brianchandotcom/liferay-portal/pull/76261
no sign of portlet startup issues

File: modules/core/portal-bootstrap/system.packages.extra.bnd (hunk 1 above)
Comment (shuyangzhou) [reply modules/core/portal-bootstrap/system.packages.extra.bnd]: @arthurchan35 on the module side, if the modules are built by our toolchain, it wiil be using bnd's "-contract: JavaPortlet". The net result is making the module's import package includes "javax.portlet" without version. This is on purpose to make the module can bind with any version that is exported from portal. You can see https://issues.liferay.com/browse/LPS-81635 for detail.

As for why exactly we are exporting 2.0, it is for backward compatibility reason, see https://issues.liferay.com/browse/LPS-81636

The reason your previous test passed, is simply because we don't have a test coverage for it.

Please take off this commit, fix the confliction and resend.
